### PR TITLE
Some CCM QRDA Fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
countable exception EncountermanagerController on refresh

### CODE REVIEW

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncountermanagerController.php [hunk 1]
```diff
@@ -78,8 +78,8 @@ public function indexAction()
                 $combination = $request->getPost('ccda_pid');
             }
 
-            for ($i = 0, $iMax = count($combination); $i < $iMax; $i++) {
-                if ($i == (count($combination) - 1)) {
+            for ($i = 0, $iMax = count($combination) ?? 0; $i < $iMax; $i++) {
```
[the remark this answers is not in the document]
Comment (sjpadgett) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncountermanagerController.php:81]: could write count($combination ?? []) however same result as the way I have.

[the remark this answers is not in the document]
Comment (sjpadgett) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncountermanagerController.php:81]: @adunsulag decided to change this as the more correct pattern would be to act on the countable argument. Still for me, either or!

**Comment by sjpadgett** [comment]
@kchapple My last commit should get us close. I'm going to test and look into ensuring our negates are correct although, I don't think you are populating reason on observation. Still the reason code should tell us and I may be able to handle in import. That is if I'm missing any.

Pretty sure we can start testing cat3 now once the cat3 header is refactored for the one forgotten boolean from ruby.

**Comment by bradymiller** [comment]
118382341-53d9d600-b5c2-11eb-8b7e-220bbab8468a

**Comment by sjpadgett** [comment]
Group vitals from individual values so we have a form for each set

image

**Comment by sjpadgett** [comment]
@kchapple 
I'll need to do some work on careplan for medications. Looks like if more than one plan medication, I only import the first.
So going to wait to retest 'til after fix later today!

**Comment by sjpadgett** [comment]
Plan of Care Activity Substance Administration

image

**Pull request merged**